### PR TITLE
Adding ROBOT template workflow with antibody example

### LINKED ISSUE #6990: New list of 49 auto-antibodies
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

Dear all,
Please find attached a new list with 49 new autoantibodies.
Many thanks,
Maud

[20210517 BrainTeam VF.xlsx](https://github.com/obophenotype/human-phenotype-ontology/files/6554953/20210517.BrainTeam.VF.xlsx)

### PR DESCRIPTION
**Pull request by matentzn**
Fixes #6990 

The PR adds a simple workflow that slurps a ROBOT template right into hp-edit.owl.

The workflow is currently implemented using a Google Sheet intermediary (which makes collaboration easier), but could just as easy use a simple TSV.

The workflow goes like this:
0. Make sure you have no uncommitted changes (!)
1. You edit the sheet. Wait for about 60 seconds for Google to propagate the updates
2. If you have previously applied the template to hp-edit.owl, and you made changes to the table, you run `make reset_edit` to return hp-edit.owl to it last committed state. Note this will _undo all changes you have made since your last commit!_ Only do this if you are sure that the only change currently in your working branch is from the template.
3. Run `sh run.sh make merge_template -B` to download the table from Google sheets and merge the terms into HPO.

Let me know if you have questions, I am also happy to run this for you, but you know, fishing rod.

Other things I did:
- Replaced 80 double whitespace occurrences with single ones.

### CODE REVIEW

**Review by pnrobinson — CHANGES_REQUESTED** [review]
So I am not sure what this PR does. It is great to add a range for robot. One question is whether the generated terms are henceforth manually editable (this would be good). Also, is there a spreadsheet where we can checkthe new definitions etc? Probably we also want to add some structure to this part of HPO but I think it would be easier to do so manually. This will not be clobbered by ROBOT later on, correct?

**Comment by matentzn** [comment]
So the way that I see it is this:

1. You edit the spreadsheet until its roughly ok. Make sure you address the points I made in the corresponding issue.
2. Then we merge in the terms as you suggest - the table is only a temporary vehicle for bulk additions.
3.  Then you go ahead and manually edit as you like, as usual!

**Comment by matentzn** [comment]
@pnrobinson this is now ready I think. I would take one last sanity look at the [diff](https://github.com/obophenotype/human-phenotype-ontology/pull/6997/files?file-filters%5B%5D=.owl), if it makes roughly sense, you can merge it and edit it on master to your hearts content.

**Review by pnrobinson — CHANGES_REQUESTED** [review]
I would suggest not making this a part of the main Makefile to avoid errors if somebody made some bad changes to the spread sheet. Can we refactor this to  make this a one-off event that would be easy to do? There are not many terms that we can easily add this way  -- the antibodies are essentially all the same but we would need more sophistication for other terms.

**Comment by matentzn** [comment]
This code is just one off code - nothing make pipeline related - I just have it in the Makefile because I that's how I know where to find it. But once this is merged the spreadsheet is forgotten! So it's not part of the Make pipeline, just a one off piece of code happening to be on Makefile

**Comment by pnrobinson** [comment]
@matentzn sorry I think I messed this up -- PRs are not great for checking changes. Could you please do a new PR to the current version of hp-edit?

**Comment by matentzn** [comment]
Not necessary, was a minor conflict i solved manually!
